### PR TITLE
Implement rendering of sl page in a alias block.

### PR DESCRIPTION
**Pull request by maethu**
Looks not too bad otherwise 😊 

Screen Shot 2020-06-04 at 17 03 20

Screen Shot 2020-06-04 at 17 03 30

- I added a `assert_html` helper in order to have a `udiff` output if it differs.

### CODE REVIEW

File: ftw/simplelayout/tests/test_alias_block.py [hunk 1]
```diff
@@ -27,6 +28,17 @@ def setUp(self):
 
         self.intids = getUtility(IIntIds)
 
+    def assert_html(self, page_html, aliasblock_html):
+        diff = difflib.unified_diff(
```
Comment (Nachtalb) [inline ftw/simplelayout/tests/test_alias_block.py:32]: According to https://docs.python.org/2.7/library/difflib.html#difflib.unified_diff, this function is for creating a diff between two lists. It seems as if you give it full strings. Wouldn't this then diff every single character to one another, meaning if one word is already different the diff would get quite big?

Comment (maethu) [reply ftw/simplelayout/tests/test_alias_block.py:32]: Good point I'll have a look, I did work on my end when I changed something in the middle. But indeed the api suggest something else 🤔

Comment (Nachtalb) [reply ftw/simplelayout/tests/test_alias_block.py:32]: If it is a problem I think you could just split the string by newlines and it would be ok.

Comment (maethu) [reply ftw/simplelayout/tests/test_alias_block.py:32]: @Nachtalb That was a really good call. I only tested some white spaces... and they worked. 

Before:
Screen Shot 2020-06-09 at 11 41 07

After:
Screen Shot 2020-06-09 at 11 43 35

Comment (Nachtalb) [reply ftw/simplelayout/tests/test_alias_block.py:32]: 🎉

**Pull request merged**